src/python/turicreate/test/test_drawing_classifier.py

```diff
@@ -305,10 +303,19 @@ def test_export_coreml_with_predict(self):
                 prefix = "pretrained" + str(test_number)
             else:
                 prefix = "scratch" + str(test_number)
-            filename = _mkstemp(prefix + ".mlmodel")[1]
+            # filename = _mkstemp(prefix + ".mlmodel")[1]
+            filename = "/Users/guihaoliang/tc_coreml.mlmodel"
             model.export_coreml(filename)
             mlmodel = _coremltools.models.MLModel(filename)
             tc_preds = model.predict(sf)
+            print(tc_preds)
```

Comment:
Generally you want to remove print statements from unit tests.

Reply:
yeah, I was cleaning the debug scripts